app/assets/javascripts/application.js

```diff
@@ -11,7 +11,6 @@
 // about supported directives.
 //
```

Comment:
The line below imports a certain version of jquery.  By adding both the line below and the script tag it imports two different versions of jquery and I'm pretty sure one version arbitrarily wins (and it may mess other stuff up too).